clients/admin-ui/src/types/api/models/ExperienceConfigCreate.ts

```diff
@@ -33,11 +35,11 @@ export type ExperienceConfigCreate = {
   /**
    * Overlay and Privacy Center 'Privacy policy link label'
    */
-  privacy_policy_link_label?: string;
+  privacy_policy_link_label?: string | null;
```

Comment:
NOTE @allisonking: I explicitly added `null` here since there's quite a different in a `PATCH` API between `undefined` and `null`, and I wanted to provide _null_ specifically.

I feel like this lil' tweak will get removed when we regenerate these types automatically...?

Reply:
hmm yes it'd get removed 🤔 so does a `privacy_policy_link_label = null` mean it's cleared, while if `privacy_policy_link_label` isn't provided, it's not touched? if so, I think we need to specify on the BE that this can be null in order for the FE types to be okay

Reply:
Yeah, since the `PATCH` semantics are "don't touch values that aren't included at all", you can do things like this:
```
{
  title: "Just update title"
}
```

...and leave everything else out. That's a subtle mismatch between the type system. 

For example, in Python the field is `privacy_policy_link_label: Optional[str]`, which seems to be converted to `privacy_policy_link_label?: string` in Typescript. Those _look_ the same and function like that in practice but there's a difference!

Reply:
oh okay, this jogged a very dormant part of my memory and I remember talking about this before: https://github.com/ethyca/fides/issues/1169

Reply:
Updated to use explicit types in the mutation code that calls the API, instead of the API types. This feels like the best compromise - it uses the autogenerated types for 90% of the way, then hacks them up a bit to reflect the "reality" of how the API is used...